pages/design/atomic-template-empty.md

```diff
@@ -41,42 +44,42 @@ _There are no major accessibility support issues known for this rule._
 
 ### Passed
 
-#### Passed example 1
+#### Passed Example 1
 
 Description...
```

Comment:
Should this be a comment?

Reply:
See - https://github.com/act-rules/act-rules.github.io/pull/488/files#r284186950